lib/rubocop/cop/flexport/engine_api_boundary.rb

```diff
@@ -141,6 +148,7 @@ module Flexport
       class EngineApiBoundary < Cop
         include EngineApi
         include EngineNodeContext
+        include FactoryBotMixin
```

Comment:
nit: let's remove the "mixin" suffix? seems non-idiomatic

https://github.com/rubocop-hq/rubocop/tree/master/lib/rubocop/cop/mixin

perhaps "FactoryBotInstantiation" or "FactoryBotUsage" if we need some kind of suffix to avoid a collision?